fbx_document.cpp

```diff
@@ -1080,15 +1083,52 @@ Error FBXDocument::_parse_materials(Ref<FBXState> p_state) {
 		Dictionary material_extensions;
 
 		if (fbx_material->pbr.base_color.has_value) {
-			material->set_albedo(_material_color(fbx_material->pbr.base_color, fbx_material->pbr.base_factor));
+			Color albedo = _material_color(fbx_material->pbr.base_color, fbx_material->pbr.base_factor);
+			material->set_albedo(albedo);
+		}
+
+		if (fbx_material->features.double_sided.enabled) {
+			material->set_cull_mode(BaseMaterial3D::CULL_DISABLED);
 		}
 
 		const ufbx_texture *base_texture = _get_file_texture(fbx_material->pbr.base_color.texture);
 		if (base_texture) {
-			{
-				bool wrap = base_texture->wrap_u == UFBX_WRAP_REPEAT && base_texture->wrap_v == UFBX_WRAP_REPEAT;
-				material->set_flag(BaseMaterial3D::FLAG_USE_TEXTURE_REPEAT, wrap);
-				material->set_texture(BaseMaterial3D::TEXTURE_ALBEDO, _get_texture(p_state, FBXTextureIndex(base_texture->file_index), TEXTURE_TYPE_GENERIC));
+			bool wrap = base_texture->wrap_u == UFBX_WRAP_REPEAT && base_texture->wrap_v == UFBX_WRAP_REPEAT;
+			material->set_flag(BaseMaterial3D::FLAG_USE_TEXTURE_REPEAT, wrap);
+			Ref<Texture2D> albedo_texture = _get_texture(p_state, FBXTextureIndex(base_texture->file_index), TEXTURE_TYPE_GENERIC);
+			if (fbx_material->fbx.transparency_color.texture_enabled) {
+				const ufbx_texture *fbx_transparency_texture = _get_file_texture(fbx_material->fbx.transparency_color.texture);
+				Ref<Texture2D> transparency_texture = _get_texture(p_state, FBXTextureIndex(fbx_transparency_texture->file_index), TEXTURE_TYPE_GENERIC);
+				if (transparency_texture.is_valid()) {
+					Ref<Image> transparency_image = transparency_texture->get_image();
+					transparency_image->decompress();
+					transparency_image->resize(albedo_texture->get_width(), albedo_texture->get_height(), Image::INTERPOLATE_LANCZOS);
+					Ref<Image> albedo_image = albedo_texture->get_image();
+					albedo_image->decompress();
```

Comment:
Ugly decompress and recompress.

Reply:
I don't think there's any way around that unfortunately, as we need the raw pixel values to decode the alpha.

Reply:
Turns out getting the original source image is not feasible as it may be loaded with an external resource loader, so we only have access to the already compressed resource.